project_stage_closed/models/__init__.py

```diff
@@ -1,2 +0,0 @@
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```

Comment:
I think you should strip from main `__init__.py` the `from . import models` clause.

Reply:
I've remove folder and init file